pa11y/lib/general.ts

```diff
@@ -0,0 +1,11 @@
+import { WireMockRestClient } from "wiremock-rest-client";
+import { Options } from "wiremock-rest-client/dist/model/options.model";
+
+export const matchGraphqlQuery = (id: string) =>
+  `$.[?(@.query=~/.*query ${id}\\(.*/s)]`;
+
+export const wiremock = (baseUri?: string, options?: Options) => {
+  return new WireMockRestClient("http://wiremock", options);
```

Comment:
Consider if the wiremock host should be defined as an environment variable instead of hardcoded.

Reply:
👍